### ISSUE TITLE
Quoted text produces an error

### ISSUE THREAD

**Issue by clnpc** [KerberX/PrivateProfileRedirector#4]
## The problem:
If a value of an option contains text enclosed in quotation marks, an error will be generated.
## Details:
Let's assume that the Fallout4.ini file contains the following line:
```
[General]
...
sStartingConsoleCommand=bat "batches\autoexec.txt"
```
and the file autoexec.txt (in <Fallout 4 Root dir>\batches subfolder) contains the following code:
`coc ctest`

Then, when the game starts without PrivateProfileRedirector being active, you (as a default PC char) will be teleported to the ctest cell and the console output will show these lines:
```
bat "batches\autoexec.txt"
Command: bat "batches\autoexec.txt"
> coc ctest
```

But if the game starts with PrivateProfileRedirector, you won't be teleported anywhere and the console window will show these lines:
```
bat "batches\autoexec.txt
Command: bat "batches\autoexec.txt
Syntax Error. Mismatched quotes.
You must enter a filename.
```

**Comment by Karandra** [comment]
Thanks. I'll try to get back to this project sometime and fix it.

**Comment by Karandra** [comment]
I've implemented the fix in the new version branch feature/kxf-and-FalloutNG-update. I still have quite a few things to update and test, but this bug should hopefully be fixed. Can't give ETA on the update release though.

**Comment by Karandra** [comment]
I forgot to close this. The update was released as v0.6.0 I think, quite a long time ago actually.
